tests/unit/go/task.yaml

```diff
@@ -4,9 +4,18 @@ systems: [ubuntu-16.04-64]
 
 restore: |
     rm -rf /tmp/unit-tests
+    rm -f /etc/os-release
+    mv /etc/os-release.orig /etc/os-release
 
 execute: |
     mkdir -p /tmp/unit-tests/src/github.com/snapcore
     cp -ar $PROJECT_PATH /tmp/unit-tests/src/github.com/snapcore
     chown -R test:12345 /tmp/unit-tests
     su -l -c "cd /tmp/unit-tests/src/github.com/snapcore/snapd && GOPATH=/tmp/unit-tests ./run-checks --unit" test
+
+    mv /etc/os-release /etc/os-release.orig
```

Comment:
This is more of a `prepare:` stage to me but the actual section looks curious. Is this intentional?